cupboard_app/v3_urls.py

```diff
@@ -34,6 +35,13 @@
         ),
         name='add_set_ingredients'
     ),
+    path(
+        'user/customingredients',
+        CustomIngredientsViewSet.as_view(
+            {'post': 'create', 'delete': 'destroy'}
+        ),
+        name='add_delete_customingredient'
```

Comment:
To make it so that delete request doesn't have a body I split the urls into delete_custom_ingredient and create_custom_ingredient. Are those names alright? I couldn't keep them in the same url since the create doesn't require an ingredient to be passed in the URL.

Reply:
Splitting the urls is the right approach! For the names, we can make it like `custom_ingredient` for the general version and `specific_custom_ingredient` similar to how we do it for the user lists. The reason being, we may end up having more different types of HTTP requests using the same endpoint url and this is not limited to create/delete.